drltrace/drltrace_frontend.cpp

```diff
@@ -97,6 +97,12 @@ static droption_t<bool> op_print_ret_addr
 (DROPTION_SCOPE_CLIENT, "print_ret_addr", false, "Print library call's return address",
  "Print return addresses of library calls.");
 
+static droption_t<uint> op_all_args
+(DROPTION_SCOPE_CLIENT, "all_args", 2, "Number of library call arguments to print",
+ "Number of library call arguments to print. Specify 0 to disable. Drltrace will print "
+ "symbolic representation of arguments in case of known library call including all "
+ "known arguments.");
```

Comment:
The description is confusing: so if the actual arguments are known, they will all be printed and this option ignored, right?  So then this should option should be named "num_unknown_args" or something, right?  And the short description should have "unknown" in it.